src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -939,12 +939,21 @@ Namespace Microsoft.CodeAnalysis.Operations
             Return New LazyParameterInitializer(parameter, value, kind, _semanticModel, syntax, type, constantValue, isImplicit)
         End Function
 
-        Private Function CreateBoundRValuePlaceholderOperation(boundRValuePlaceholder As BoundRValuePlaceholder) As IPlaceholderOperation
+        Private Function CreateBoundRValuePlaceholderOperation(boundRValuePlaceholder As BoundRValuePlaceholder) As IOperation
             Dim syntax As SyntaxNode = boundRValuePlaceholder.Syntax
             Dim type As ITypeSymbol = boundRValuePlaceholder.Type
             Dim constantValue As [Optional](Of Object) = ConvertToOptional(boundRValuePlaceholder.ConstantValueOpt)
             Dim isImplicit As Boolean = boundRValuePlaceholder.WasCompilerGenerated
-            Return New PlaceholderExpression(_semanticModel, syntax, type, constantValue, isImplicit)
+            If syntax.IsKind(SyntaxKind.ConditionalAccessExpression) Then
```

Comment:
>If syntax.IsKind(SyntaxKind.ConditionalAccessExpression) Then [](start = 12, length = 61)

The syntax check is fine for now, but we should come up with a more robust way of detecting conditional access receivers. Either keep the original issue open, or open a new one.

Reply:
For example, we could try creating node for the placeholder when we handle BoundConditionalAccess in CreateBoundConditionalAccessOperation. The operation will be cached and this method will never be called for this bound node.

---
In reply to: [149583274](https://github.com/dotnet/roslyn/pull/23068#discussion_r149583274) [](ancestors = 149583274)